src/Orchard.Specs/Numeric.feature

```diff
@@ -106,29 +105,29 @@ Scenario: Creating and using numeric fields
         And I hit "Save Draft"
     Then I should see "Guests is an invalid number"
 
-    # The default value should be proposed on creation
-    When I go to "Admin/ContentTypes/Edit/Event"
-        And I fill in
-            | name                                        | value |
-            | Fields[0].NumericFieldSettings.DefaultValue | 1234  |
-        And I hit "Save"
-        And I go to "Admin/Contents/Create/Event"
-    Then I should see "value=\"1234\""
+    ## The default value should be proposed on creation
```

Comment:
Removed?

Reply:
This comment is test about Default value should be proposed on creation.
But when I go to "Admin/ContentTypes/Edit/Event" and look at Numeric Field. It has no text box of Default value for input data.

Reply:
@sebastienros I think _Default value` was removed by accident after you merged `1.10.x` branch to `dev` ? see: commit: https://github.com/OrchardCMS/Orchard/commit/c8a4def111065c5bb2039638ce9507604ab51e55

Reply:
@BenedekFarkas do you remember the state of the Default value? I know it was adding some bugs at some point. Is it really just a merge issue?

Reply:
As far as I remember, that's working OK for all the built-in fields.
[Here is the latest SpecFlow test output for this test on 1.10.x](https://teamcity.jetbrains.com/viewLog.html?buildId=1643001&buildTypeId=Orchard_MinorVersion_BuildAndSpecFlowTests&tab=buildLog#_state=23603,23610,30720).

Reply:
@sebastienros @BenedekFarkas But in this PR, We working on dev branch.